src/User32.Desktop/User32.cs

```diff
@@ -109,6 +128,12 @@ public static partial class User32
         [DllImport(nameof(User32))]
         public static extern int SendMessage(IntPtr hWnd, int wMsg, IntPtr wParam, IntPtr lParam);
 
+        [DllImport(nameof(User32))]
+        public static extern int SendMessage(IntPtr hWnd, int wMsg, IntPtr wParam, StringBuilder lParam);
```

Comment:
ok. Well, given the variety of types that can be passed into `SendMessage`, I think @vbfox is right: the caller should just pin the buffer and send the pointer manually rather than try to add an overload for every combination of parameter types.
Unless you want to pursue the generic type argument method variety he suggested.